ADALiOS/ADALiOS/ADAuthenticationResult.h

```diff
@@ -54,14 +54,15 @@ typedef enum
  case of error.*/
 @property (readonly) NSString* accessToken;
 
+/*! A valid access token, if the results indicates success. The property is
+ calculated from the tokenCacheStoreItem one. The property is nil, in
+ case of error.*/
+@property (readonly) NSString* idToken;
```

Comment:
This was a prototype property that can be removed. I now just assign the idToken to the accessToken if we are in a flow for idToken only.

removed.